docker-compose.yaml

```diff
@@ -0,0 +1,88 @@
+services:
```

Comment:
I am getting a port error when running `docker-compose up`. Could be something on my end though, not sure.

```
[+] Running 6/5
 ✔ Network project-zeno_default              Created                                                                                0.0s 
 ✔ Volume "project-zeno_database_data"       Created                                                                                0.0s 
 ✔ Container project-zeno-api-1              Created                                                                                0.1s 
 ✔ Container project-zeno-frontend-1         Created                                                                                0.1s 
 ✔ Container project-zeno-db-1               Created                                                                                0.1s 
 ✔ Container project-zeno-langfuse-server-1  Created                                                                                0.0s 
Attaching to api-1, db-1, frontend-1, langfuse-server-1
Gracefully stopping... (press Ctrl+C again to force)
Error response from daemon: driver failed programming external connectivity on endpoint project-zeno-db-1 (cb3d16eb9595ee17601e03ec769e8faa92ff443fc77e7b1cf2493f2132ec6b56): Bind for 0.0.0.0:5432 failed: port is already allocated
```